inductiva/_benchmarks/benchmark.py

```diff
@@ -120,7 +121,7 @@ def run(self, num_repeats: int = 2) -> Self:
         """
         with self:
             for simulator, input_dir, machine_group, kwargs in self.runs:
-                machine_group.start(wait_on_pending_quota=True)
+                machine_group.start(wait_on_pending_quota=False)
```

Comment:
Was this changed just for testing?

Reply:
Actually, I intended for this value to be set to ```False``` from the beginning. I think it makes more sense for the initial version to crash if the user runs out of quotas, rather than causing the entire benchmark to stall while waiting for more quotas. In future versions, parameters like this will be configurable.